frontend/src/scenes/sessions/sessionsPlayLogic.ts

```diff
@@ -79,4 +79,15 @@ export const sessionsPlayLogic = kea<sessionsPlayLogicType<SessionPlayerData>>({
             },
         },
     }),
+    selectors: {
+        sessionTimestamp: [
+            (selectors) => [selectors.sessionPlayerData],
+            (sessionPlayerData: SessionPlayerData): string | null => {
+                if (!sessionPlayerData?.snapshots.length || !sessionPlayerData.snapshots[0].timestamp) {
+                    return null
+                }
+                return moment(sessionPlayerData.snapshots[0].timestamp).format('lll')
```

Comment:
This is (dangerously) wrong as this is client time. We do have access to the correct timestamp, but exposing it requires server-side work.

Reply:
Thanks for the quick heads up! I've hidden this for now until further review. I do think that this might be good enough, as we're not showing exact time, a small discrepancy on client's clock should be fine. There is also the issue that the server side timestamp might not be the exact time either due to factors such as network latency (and if we re-enable batching in the future).